src/client/src/actions/wizardContentActions/parseFrameworksPayload.ts

```diff
@@ -1,38 +1,21 @@
-import EngineAPIService from "../../services/EngineAPIService";
 import getSvgUrl from "../../utils/getSvgUrl";
 import { IMetadata } from "../../types/metadata";
 import { IOption } from "../../types/option";
+import { FRAMEWORK_TYPE } from "../../utils/constants";
 
-type FrameworkType = "frontend" | "backend";
-
-// thunk
-export const getFrameworks = async (
-  projectType: string,
-  type: FrameworkType,
-  isPreview: boolean,
-  serverPort: number
-): Promise<IOption[]> => {
-  const api = new EngineAPIService(serverPort, undefined);
-  try {
-    const frameworksJson = await api.getFrameworks(projectType);
-
-    if (frameworksJson.detail == null) {
-      return getOptionalFromMetadata(
-        getMetadataFromJson(frameworksJson, type, isPreview)
-      );
-    } else {
-      console.log("FAILED");
-      return [];
-    }
-  } catch (error) {
-    console.log(error);
-    return [];
-  }
+export const parseFrameworksPayload = (
```

Comment:
Probably, we should move this file to the utils folder since these functions are not actions but helper functions.

Reply:
PR updated